automation_tools/__init__.py

```diff
@@ -820,7 +815,7 @@ def setup_python_code_coverage():
     if not epel_present:
         run('yum -y install https://dl.fedoraproject.org/pub/epel/'
```

Comment:
direct package install will work I suppose

Reply:
Yes it works!